app/services/foreman_ansible/operating_system_parser.rb

```diff
@@ -55,6 +55,8 @@ def os_major
           (facts[:ansible_distribution_major_version][%r{\/sid}i] ||
            facts[:ansible_distribution_major_version] == 'n/a')
         debian_os_major_sid
+      elsif os_name == 'Ubuntu'
+        facts[:ansible_distribution_major_version]
```

Comment:
Shouldn't the `else` clause capture this already?

Reply:
Thanks :see_no_evil: